### PR TITLE
chagall: fix regulators in dts

### PR DESCRIPTION
**Pull request by clamor-s**
minor change in chagall power set according to downstream

### CODE REVIEW

**Comment by digetx** [comment]
Thanks!

File: arch/arm/boot/dts/tegra30-pegatron-chagall.dts [hunk 1]
```diff
@@ -1416,6 +1416,17 @@
 		vin-supply = <&vdd_3v3_sys>;
 	};
 
+	vdd_5v0_bl: regulator-bl {
+		compatible = "regulator-fixed";
+		regulator-name = "vdd_5v0_bl";
+		regulator-min-microvolt = <5000000>;
+		regulator-max-microvolt = <5000000>;
+		regulator-boot-on;
+		gpio = <&gpio TEGRA_GPIO(C, 6) GPIO_ACTIVE_HIGH>;
```
Comment (digetx) [inline arch/arm/boot/dts/tegra30-pegatron-chagall.dts:1425]: Could you please give a reference to the downstream code?

Comment (clamor-s) [reply arch/arm/boot/dts/tegra30-pegatron-chagall.dts:1425]: https://github.com/sonntam/android_kernel_fujitsu_mx532/blob/master/arch/arm/mach-tegra/board-chagall-power.c#L574

Comment (digetx) [reply arch/arm/boot/dts/tegra30-pegatron-chagall.dts:1425]: Thanks

**Pull request merged**